ui/.eslintrc.js

```diff
@@ -48,7 +48,8 @@ module.exports = {
   },
   plugins: ["@typescript-eslint", "babel", "prettier", "react", "standard"],
   rules: {
-    "react/jsx-no-bind": "error",
+    "react/display-name": "off",
```

Comment:
eslint errors where giving me more trouble than they were helpful.

In particular, `display-name` disallowed me of having small helper components, like `WrapWithIf` (great name, i know)